lib/services/local/users.go

```diff
@@ -676,3 +787,56 @@ func (s *IdentityService) GetOIDCAuthRequest(stateToken string) (*services.OIDCA
 	}
 	return req, nil
 }
+
+func (s *IdentityService) createAndUpsertTOTP(user string) (otpURI string, otpQRCode []byte, err error) {
```

Comment:
same here. Our `services` package should be renamted to `model` packages as they are really just CRUD logic.